MobiFlight/Joystick.cs

```diff
@@ -392,16 +392,18 @@ private String GetAxisNameForUsage(int usage)
             return UsageMap[usage];
         }
 
-        public void SetOutputDeviceState(string name, string value)
+        public void SetOutputDeviceState(string name, byte state)
         {
-            byte state = byte.Parse(value);
+            double tempValue;
 
             foreach(var light in Lights)
             {
                 if (light.Label != name) continue;
+
+                state = (byte)(state > 0 ? 1 : 0);
```

Comment:
Why not have this take a Bool instead and demand the callers provide the correctly formatted data in the first place? Then you can just cast it to a byte without all the other stuff. The only place this is called now anyway is ExecutionManager, and that's already explicitly only sending 0 or 1.

Reply:
yes. true. With that reasoning in mind i made the change of the function signature and I am providing a byte now. I will remove it.